lib/src/index.js

```diff
@@ -372,6 +375,93 @@ function initialize(element) {
             detail: event.detail,
         };
 
+    }
+
+    function verifyClickCell(cell)
+    {
+        const mouseDownCell = context.mouseDownCell;
+        if (context.state.resizableColumn || context.state.resizableRow)
+            return false;
+        if (context.didReorder)
+            return false;
+        if (cell === null)
+            return false;
+        if (stringifyId(cell.columnId) !== stringifyId(mouseDownCell.columnId))
+            return false;
+        if (stringifyId(cell.rowId) !== stringifyId(mouseDownCell.rowId))
+            return false;
+
+        return true;
+    }
+
+    function buildClickEventProperties(event)
+    {
+        return {
+            ctrlKey: event.ctrlKey,
+            shiftKey: event.shiftKey,
+            button: event.button,
+            buttons: event.buttons,
+            detail: event.detail,
+        };
+
+    }
+
+    context.addEventListener(element, 'contextmenu', (event) => {
+        // Do not forward context menu events whilst editing cells
+        if (context.state.text)
+            return;
+        updateState(context);
```

Comment:
It would be good to call it before the early return condition, as you are using the state there as well.

Theoretically it should be ok, as the `input` event already calls the `updateState` internally. But that may change in the future.